### ISSUE TITLE
Demo webpage

### ISSUE THREAD

**Issue by canonet28** [Jamee999/cricket#33]
As mentioned elsewhere, I made a few changes to the code to produce a few extra output files when running a match. Then I put all that into a basic webpage format. While it could have been more elegant and less forceful at times, I have got things working exactly as I wanted. In fact, already got the Wisden Trophy on to a cracking start!! 

For anybody curious: http://rusty-league.net/tms/

**Comment by pares49** [comment]
This doesn't seem to work any more canonet28? Any chance we can get it back up and running for Christmas!

**Comment by canonet28** [comment]
In fact, that demo webpage became a passion web project...here is the link: https://tms.rusty-league.net/

**Comment by pares49** [comment]
this is very cool.

is there a way to run custom games off it?

**Comment by canonet28** [comment]
I have run custom games based on certain themes previously. One such theme was a series of Test matches where a national team from the late 1990s played the same nation's present-day team. Was pretty fun and is available on the website under "Thematic" section.

**Comment by pares49** [comment]
I just wondered if you were able to set it up for us to run custom games off it (ie. if we put in our own XIs)? It is such a user friendly looking tool!

**Comment by canonet28** [comment]
Nope..don't know much Python. Just brute-forced my way through Jamie's codes from the little I know to produce all those extra ball-by-ball commentary files/match viewer output files..still running it off the text files locally. The website in turn uses PHP to create the final web layout and update player stats from the output files.
